### ISSUE TITLE
Synchronization Errors

### ISSUE THREAD

**Issue by colyz93** [hyyan/woo-poly-integration#515]
#### Can you reproduce this issue on default Wordpress theme (eg Storefront)?
yes
#### Can you reproduce this issue when all other plugins are disabled except WooCommerce, Polylang and Hyyan WooCommerce Polylang Integration?
yes
#### What product versions and settings are you using when this issue occurs?
* PHP: 7.4
* WordPress: 5.5.1
* WooCommerce: 4.5.2
* Polylang: 2.8.2
* Hyyan WooCommerce Polylang Integration: 1.4.3
* Browser: Google Chrome 

#### Steps to Reproduce
1. Update Product page in the first lang. ; Publish Or Update product page in the second lang.

#### What I Expected
when i publish the product page in the second i expect it to sync availability and to show stock the same as the product page in the first lang.
when i edit the page of the second language the page should run smoothly without lagging or freezing

#### What Happened Instead
i had the problem for variable products return to simple and i applied the resolve mentioned [here](https://github.com/hyyan/woo-poly-integration/issues/510) after that:
when i publish or update any of both lang. pages the second language page show as out of stock
when i publish or update any of both lang. pages the variations multiply in second language page, although they have stock the product page show out of stock
second problem when i load the second language product page it gets freezing and lagging 
these problems reproduce also with WP 5.4.2, WC 4.4.1, Hyyan 1.4.2
I'm now trying to use polylang for woocommerce combined with hyyan but it doesn't give me the expected functions and results

## WordPress Environment

<details>
```
`
### WordPress Environment ###

WordPress address (URL): https://www.khanalawam.com
Site address (URL): https://www.khanalawam.com
WC Version: 4.5.2
REST API Version: ✔ 4.5.2
WC Blocks Version: ✔ 3.1.0
Action Scheduler Version: ✔ 3.1.6
WC Admin Version: ✔ 1.5.0
Log Directory Writable: ✔
WP Version: 5.5.1
WP Multisite: –
WP Memory Limit: 2 GB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: Apache
PHP Version: 7.4.10
PHP Post Max Size: 2 GB
PHP Time Limit: 3000
PHP Max Input Vars: 6000
cURL Version: 7.72.0
OpenSSL/1.1.1g

SUHOSIN Installed: –
MySQL Version: 5.6.41-84.1
Max Upload Size: 2 GB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 4.5.2
WC Database Prefix: ayS_
Total Database Size: 162.84MB
Database Data Size: 143.71MB
Database Index Size: 19.13MB
ayS_woocommerce_sessions: Data: 0.21MB + Index: 0.01MB + Engine MyISAM
ayS_woocommerce_api_keys: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_woocommerce_attribute_taxonomies: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_woocommerce_downloadable_product_permissions: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_woocommerce_order_items: Data: 0.02MB + Index: 0.01MB + Engine MyISAM
ayS_woocommerce_order_itemmeta: Data: 0.15MB + Index: 0.11MB + Engine MyISAM
ayS_woocommerce_tax_rates: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_woocommerce_tax_rate_locations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_woocommerce_shipping_zones: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_woocommerce_shipping_zone_locations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_woocommerce_shipping_zone_methods: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_woocommerce_payment_tokens: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_woocommerce_payment_tokenmeta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_woocommerce_log: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_actionscheduler_actions: Data: 1.16MB + Index: 0.28MB + Engine MyISAM
ayS_actionscheduler_claims: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_actionscheduler_groups: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
ayS_actionscheduler_logs: Data: 0.57MB + Index: 0.40MB + Engine MyISAM
ayS_adtribes_my_conversions: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_aepc_custom_audiences: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_aepc_logs: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_b2s_posts: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_b2s_posts_drafts: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_b2s_posts_favorites: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_b2s_posts_network_details: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_b2s_posts_sched_details: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_b2s_user: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
ayS_b2s_user_contact: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_b2s_user_network_settings: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_berocket_termmeta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_braapf_product_stock_status_parent: Data: 0.02MB + Index: 0.03MB + Engine MyISAM
ayS_braapf_product_variation_attributes: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
ayS_braapf_term_taxonomy_hierarchical: Data: 0.01MB + Index: 0.02MB + Engine MyISAM
ayS_braapf_variation_attributes: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
ayS_cartflows_ca_cart_abandonment: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
ayS_cartflows_ca_email_history: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_cartflows_ca_email_templates: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_cartflows_ca_email_templates_meta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_commentmeta: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
ayS_comments: Data: 0.12MB + Index: 0.05MB + Engine MyISAM
ayS_css_js_manager: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_ewwwio_images: Data: 1.63MB + Index: 0.72MB + Engine MyISAM
ayS_ewwwio_queue: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_ghostmonitor_data: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
ayS_hustle_entries: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
ayS_hustle_entries_meta: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
ayS_hustle_modules: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
ayS_hustle_modules_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
ayS_hustle_tracking: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
ayS_links: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_mailchimp_carts: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_mailchimp_jobs: Data: 0.02MB + Index: 0.00MB + Engine MyISAM
ayS_mclean_refs: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_mclean_scan: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_ms_snippets: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_nextend2_image_storage: Data: 0.01MB + Index: 0.00MB + Engine MyISAM
ayS_nextend2_section_storage: Data: 0.05MB + Index: 0.01MB + Engine MyISAM
ayS_nextend2_smartslider3_generators: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_nextend2_smartslider3_sliders: Data: 0.06MB + Index: 0.00MB + Engine MyISAM
ayS_nextend2_smartslider3_sliders_xref: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_nextend2_smartslider3_slides: Data: 0.04MB + Index: 0.01MB + Engine MyISAM
ayS_nxs_query: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_options: Data: 10.16MB + Index: 0.40MB + Engine MyISAM
ayS_pmxi_files: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_pmxi_history: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_pmxi_images: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_pmxi_imports: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_pmxi_posts: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_pmxi_templates: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_postmeta: Data: 68.86MB + Index: 3.25MB + Engine MyISAM
ayS_posts: Data: 15.07MB + Index: 0.29MB + Engine MyISAM
ayS_premmerce_bundles: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_premmerce_bundles_products: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_rank_math_404_logs: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
ayS_rank_math_internal_links: Data: 0.02MB + Index: 0.01MB + Engine MyISAM
ayS_rank_math_internal_meta: Data: 0.02MB + Index: 0.02MB + Engine MyISAM
ayS_rank_math_redirections: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_rank_math_redirections_cache: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
ayS_rank_math_sc_analytics: Data: 0.99MB + Index: 0.29MB + Engine MyISAM
ayS_snippets: Data: 0.01MB + Index: 0.00MB + Engine MyISAM
ayS_stock_log: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
ayS_termmeta: Data: 0.25MB + Index: 0.19MB + Engine MyISAM
ayS_terms: Data: 0.16MB + Index: 0.20MB + Engine MyISAM
ayS_term_relationships: Data: 0.30MB + Index: 0.55MB + Engine MyISAM
ayS_term_taxonomy: Data: 1.31MB + Index: 0.11MB + Engine MyISAM
ayS_tinvwl_analytics: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_tinvwl_items: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_tinvwl_lists: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_tm_taskmeta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_tm_tasks: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_UPCP_Catalogues: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_UPCP_Catalogue_Items: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_UPCP_Categories: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_UPCP_Custom_Fields: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_UPCP_Fields_Meta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_UPCP_Items: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_UPCP_Item_Images: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_UPCP_SubCategories: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_UPCP_Tagged_Items: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_UPCP_Tags: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_UPCP_Tag_Groups: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_UPCP_Videos: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_usermeta: Data: 0.44MB + Index: 0.15MB + Engine MyISAM
ayS_users: Data: 0.01MB + Index: 0.02MB + Engine MyISAM
ayS_wcpdf_invoice_number: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_wc_admin_notes: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_wc_admin_note_actions: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_wc_category_lookup: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_wc_customer_lookup: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
ayS_wc_download_log: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_wc_order_coupon_lookup: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_wc_order_product_lookup: Data: 0.01MB + Index: 0.02MB + Engine MyISAM
ayS_wc_order_stats: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
ayS_wc_order_tax_lookup: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_wc_product_meta_lookup: Data: 0.07MB + Index: 0.09MB + Engine MyISAM
ayS_wc_reserved_stock: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_wc_tax_rate_classes: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
ayS_wc_webhooks: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_wfblockediplog: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
ayS_wfblocks7: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
ayS_wfconfig: Data: 0.45MB + Index: 0.00MB + Engine InnoDB
ayS_wfcrawlers: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
ayS_wffilechanges: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
ayS_wffilemods: Data: 13.55MB + Index: 0.00MB + Engine InnoDB
ayS_wfhits: Data: 0.05MB + Index: 0.05MB + Engine InnoDB
ayS_wfhoover: Data: 1.52MB + Index: 0.16MB + Engine InnoDB
ayS_wfissues: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
ayS_wfknownfilelist: Data: 5.52MB + Index: 0.00MB + Engine InnoDB
ayS_wflivetraffichuman: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
ayS_wflocs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
ayS_wflogins: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
ayS_wfls_2fa_secrets: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
ayS_wfls_settings: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
ayS_wfnotifications: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
ayS_wfpendingissues: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
ayS_wfreversecache: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
ayS_wfsnipcache: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
ayS_wfstatus: Data: 0.42MB + Index: 0.22MB + Engine InnoDB
ayS_wftrafficrates: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
ayS_woobe_history: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
ayS_woobe_history_bulk: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
ayS_woocommerce_custom_region_by_country: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
ayS_woocommerce_gpf_google_taxonomy: Data: 0.89MB + Index: 0.00MB + Engine MyISAM
ayS_woo_ua_auction_log: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_woo_wallet_transactions: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
ayS_woo_wallet_transaction_meta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_wow_coder: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
ayS_wpforms_tasks_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
ayS_wsal_metadata: Data: 18.27MB + Index: 9.98MB + Engine MyISAM
ayS_wsal_occurrences: Data: 0.69MB + Index: 0.77MB + Engine MyISAM
ayS_wsal_options: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
ayS_ywrr_email_blocklist: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
ayS_ywrr_email_schedule: Data: 0.02MB + Index: 0.00MB + Engine MyISAM

### Post Type Counts ###

amp_validated_url: 49
attachment: 1142
br_filters_group: 2
br_product_filter: 22
cmm4e_menu: 1
cmm4e_menu_theme: 1
custom_css: 5
customize_changeset: 2
elementor_library: 1
fpf_fields: 1
gfw_report: 35
hollerbox: 1
is_search_form: 5
mc4wp-form: 1
nav_menu_item: 65
page: 29
polylang_mo: 2
popup: 1
popup_theme: 7
post: 2
product: 971
product_variation: 182
revision: 2
rvp: 1
shop_coupon: 5
shop_order: 110
size-chart: 15
soliloquy: 1
wcps: 1
wcps_layout: 1
woo_discount_cart: 1
wpforms: 2
wpmm_theme: 3
wpwautoposterlogs: 368
wrsl: 1
yith_wcps_type: 1
yith-wcbm-badge: 2

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (59) ###

Query Monitor: by John Blackbourn – 3.6.4
Polylang: by WP SYNTEX – 2.8.2
User Menus: by Code Atlantic – 1.2.4
Add Polylang support for Customizer: by richardev – 1.3.6
AJAX Search for WooCommerce: by Damian Góra – 1.7.2 – Installed version not tested with active version of WooCommerce 4.5.2
Beeketing For WooCommerce: by Beeketing – 3.8.16 – Installed version not tested with active version of WooCommerce 4.5.2
EditorsKit: by Jeffrey Carandang – 1.29.3
Brands for WooCommerce: by BeRocket – 3.5.1.9
Child Theme Configurator: by Lilaea Media – 2.5.6
Code Snippets: by Code Snippets Pro – 2.14.0
Custom Sidebars: by WPMU DEV – 3.2.3
Customer Reviews for WooCommerce: by Customer Reviews – 3.125
Easy Table of Contents: by Steven A. Zahm – 2.0.11
EWWW Image Optimizer: by Exactly WWW – 5.7.1
Exit Popup: by Brontobytes – 3.1
Instant Indexing: by Rank Math – 1.0.0
Site Kit by Google: by Google – 1.16.0
Jetpack by WordPress.com: by Automattic – 8.9.1
Kadence Blocks – Gutenberg Blocks for Page Builder Features: by Kadence WP – 1.8.14
WooCommerce Load More Products: by BeRocket – 1.1.7.4
Mailchimp for WooCommerce: by Mailchimp – 2.4.5 – Installed version not tested with active version of WooCommerce 4.5.2
MC4WP: Mailchimp for WordPress: by ibericode – 4.8.1
MOJO Marketplace: by Mike Hansen – 1.5.7
Official Facebook Pixel: by Facebook – 2.2.0
WooCommerce Onsale Page: by wpgenie – 1.0.10 – Installed version not tested with active version of WooCommerce 4.5.2
OneSignal Push Notifications: by OneSignal – 2.1.4
Premmerce Frequently Bought Together for WooCommerce: by Premmerce – 1.0.9 – Installed version not tested with active version of WooCommerce 4.5.2
Product GTIN (EAN, UPC, ISBN) for WooCommerce: by Emanuela Castorina – 1.0.9 – Installed version not tested with active version of WooCommerce 4.5.2
QuadMenu: by Mega Menu – 2.0.4
Rank Math SEO: by Rank Math – 1.0.47.1
Smart Image Resize for WooCommerce: by Nabil Lemsieh – 1.4.1 – Installed version not tested with active version of WooCommerce 4.5.2
Smart Slider 3: by Nextend – 3.4.1.9
Super Socializer: by Team Heateor – 7.12.50
Theme and plugin translation for Polylang (TTfP): by Marcin Kazmierski – 3.2.4
TI WooCommerce Wishlist: by TemplateInvaders – 1.21.7
UpdraftPlus - Backup/Restore: by UpdraftPlus.Com
DavidAnderson – 1.16.29

Advanced Product Size Charts for WooCommerce Premium: by theDotstore – 2.0.4 – Installed version not tested with active version of WooCommerce 4.5.2
WOOBE - WooCommerce Bulk Editor and Products Manager: by realmag777 – 1.0.6.1
WooCommerce Cart Abandonment Recovery: by CartFlows Inc – 1.2.8 – Installed version not tested with active version of WooCommerce 4.5.2
Checkout Field Editor for WooCommerce: by ThemeHigh – 1.4.3 – Installed version not tested with active version of WooCommerce 4.5.2
WooCommerce Order Cancel For Customers: by WPManageNinja – 1.1 – Installed version not tested with active version of WooCommerce 4.5.2
Hyyan WooCommerce Polylang Integration: by Hyyan Abo Fakher – 1.4.3 – Installed version not tested with active version of WooCommerce 4.5.2
Product Feed PRO for WooCommerce: by AdTribes.io – 8.7.3
Woocommerce Single Product Page Customizer: by Geek Web Solution – 1.3 – Installed version not tested with active version of WooCommerce 4.5.2
Additional Variation Images Gallery for WooCommerce: by Emran Ahmed – 1.1.42
Variation Swatches for WooCommerce: by Emran Ahmed – 1.0.86 – Installed version not tested with active version of WooCommerce 4.5.2
TeraWallet: by WCBeginner – 1.3.16 – Installed version not tested with active version of WooCommerce 4.5.2
Advanced AJAX Product Filters for WooCommerce: by BeRocket – 1.5.1.4
WooCommerce Colors: by WooThemes – 1.0.7 – Installed version not tested with active version of WooCommerce 4.5.2
WooCommerce Egypt Cities: by Mohamed Mounir – 2.0.0 – Installed version not tested with active version of WooCommerce 4.5.2
WooCommerce Extra Product Sorting Options: by SkyVerge – 2.8.4 – Installed version not tested with active version of WooCommerce 4.5.2
WooCommerce Google Analytics Integration: by WooCommerce – 1.4.23 – Installed version not tested with active version of WooCommerce 4.5.2
Booster for WooCommerce: by Pluggabl LLC – 5.3.2 – Installed version not tested with active version of WooCommerce 4.5.2
WooCommerce Product Image Flipper: by jameskoster – 0.4.2 – Installed version not tested with active version of WooCommerce 4.5.2
WooCommerce: by Automattic – 4.5.2
Wordfence Security: by Wordfence – 7.4.11
Asset CleanUp: Page Speed Booster: by Gabriel Livan – 1.3.6.9
WP Rocket: by WP Media – 3.7.1.1
WPFront Scroll Top: by Syam Mohan – 2.0.2

### Inactive Plugins (0) ###


### Dropin Plugins (2) ###

advanced-cache.php: advanced-cache.php
db.php: Query Monitor Database Class

### Must Use Plugins (2) ###

Endurance Page Cache: by Mike Hansen – 2.0.2
SSO: by Garth Mortensen
Mike Hansen – 0.3


### Settings ###

API Enabled: –
Force SSL: –
Currency: EGP (EGP)
Currency Position: right_space
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: auction (auction)
external (external)
grouped (grouped)
simple (simple)
variable (variable)
wooco (wooco)
woosb (woosb)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔

### WC Pages ###

Shop base: #7 - /shop/
Cart: #8 - /cart/
Checkout: #9 - /checkout/
My account: #10 - /my-account/
Terms and conditions: #365 - /terms-and-conditions/

### Theme ###

Name: Khan Al-Awam
Version: 1.0.1599921027
Author URL: 
Child Theme: ✔
Parent Theme Name: Neve
Parent Theme Version: 2.8.1
Parent Theme Author URL: https://themeisle.com
WooCommerce Support: ✔

### Templates ###

Overrides: /home1/khanalaw/public_html/wp-content/plugins/woo-variation-gallery/templates/product-images.php
/home1/khanalaw/public_html/wp-content/plugins/woo-variation-gallery/templates/product-thumbnails.php
khanalawam/woocommerce/single-product/rating.php


### TI WooCommerce Wishlist Templates ###

Overrides: –

### Action Scheduler ###

Complete: 3,426
Oldest: 2020-09-06 05:01:24 +0200
Newest: 2020-09-19 13:02:24 +0200

Failed: 2
Oldest: 2020-01-07 11:22:29 +0200
Newest: 2020-04-05 14:12:20 +0200

Pending: 1
Oldest: 2020-09-19 13:17:24 +0200
Newest: 2020-09-19 13:17:24 +0200

````
</details>

**Comment by dani-gestionatuweb** [comment]
Hello, I have the same problem with the latest versions of WooCommerce.

**Comment by clementhadrot** [comment]
Hi, i have same problem... Suggestions ?

**Comment by flux28** [comment]
same issue. Polylang doesn't copy and synchronize the metadata such as the SKU, the price, doesn't synchronize product variation. Products in both languages are no longer linked.

**Comment by fourlife** [comment]
Not an expert, but this seems to work for me:

Downgrade the following: WP to version 5.4.2, woocommerce to 4.4.1, polylang to 2.7.4.
Install "enable jquery migrate helper" plugin.

Still have some duplicated variations, but can fix them manually. Products sync ok, and don't show false "out of stocks" when selling.

Waiting for an update or a better solution though

**Comment by javiergaeten1** [comment]
I have the same issue. Any other solution?? :(

**Comment by fourlife** [comment]
Hi all, any news on this?

My WP and plugin versions are updated to the last versions, but still have Polylang at 2.7.4. When updating it, keeps showing the false "out of stock" on one of the languages when selling a product.

**Comment by jacopobonomi** [comment]
it's very frustrating, there are so many variations in the second language that there shouldn't be. After selling a product it goes out of stock. How can we solve it? now i don't know how to change plugin because the site is already in production. Can someone help me?

**Comment by abab2020** [comment]
I had the problem with variable products showing as simple when editing. That was solved by installing version 1.4.4. I still have the problem variable products showing as out of stock  after one sale, while they are not out of stock. Saving the product again solves this, but the problem stays. Did anyone find a solution for this?

**Comment by Jon007** [comment]
Hi Variations problems are generally fixed with recent checkins to the source code, it would be helpful if some people could confirm whether that this the case.
If not open separate github issues with more detail for any remaining issue.
If it is ok then we can ask Hyyan to do a release,
thanks
